tests/test_config.py

```diff
@@ -746,10 +759,14 @@ def test_specifiers(tmp_path: Path) -> None:
             "ImageVersion": "1.2.3",
             "OutputDirectory": str(Path.cwd() / "abcde"),
             "Output": "test",
+            "TestValue": "100"
         }
 
         assert {k: v for k, v in config.environment.items() if k in expected} == expected
 
+        expected_env_files = [Path.cwd() / "mkosi.env", Path.cwd() / "other.env"]
+        assert config.environment_files == expected_env_files
```

Comment:
Please make a new test `test_environment()` for this instead of putting it in `test_specifiers()`.